### PR TITLE
Adding React course

### PR DESCRIPTION
**Pull request by Prathamesh180**
## What does this PR do?
Add resource

### Description
Added a course on React created by Hitesh Choudhary hosted on YouTube.

### Why is this valuable (or not)?
It is very useful as he is very experienced person and teaches very well.

### How do we know it's really free?
Because he has created it as open playlist.

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course.

## Checklist:
- [ x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ x] Include author(s) and platform where appropriate.
- [ x] Put lists in alphabetical order, correct spacing.
- [ x] Add needed indications (PDF, access notes, under construction).
- [ x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
This course does not seem to be Tamil

**Comment by Prathamesh180** [comment]
This is not in tamil, it is in english, have I done anything wrong? I will do necessary changes. It is a reactjs course in english.

**Comment by Prathamesh180** [comment]
I have made necessary changes ( moved course to english section ). I have one doubt will this merge considered for hacktoberfest? I am not seeing hacktoberfest accepted tags on merges.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -967,6 +967,7 @@
 
 #### React
 
+* [Complete React course for beginner](https://youtube.com/playlist?list=PLRAV69dS1uWQos1M1xP6LWN6C-lZvpkmq&si=Mu6nucz4_62aP3V0) - Hitesh Choudhary
```
Comment (eshellman) [inline courses/free-courses-en.md:970]:
```suggestion
* [Complete React course for beginner](https://youtube.com/playlist?list=PLRAV69dS1uWQos1M1xP6LWN6C-lZvpkmq) - Hitesh Choudhary
```
entire repo is marked correctly for Hacktoberfest

Comment (Prathamesh180) [reply courses/free-courses-en.md]: I have made all seggested changes in the course link.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**